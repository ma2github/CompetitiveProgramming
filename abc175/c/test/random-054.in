-359903191321038 429626629421936 452129628720069
