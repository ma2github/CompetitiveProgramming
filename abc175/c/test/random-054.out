359903191321038
